reconcile/utils/terrascript_client.py

```diff
@@ -251,6 +262,14 @@ def download_logtoes_zip(self, release_url):
             open(zip_file, 'wb').write(r.content)
         return zip_file
 
+    def init_github(self):
```

Comment:
Type hint here, please.

Reply:
Do you know how to annotate the type of `self`?

Reply:
I don't need it for `self` but at leat the return value: `init_github(self) -> None:` - this way we get mypy to check its callees.